### ISSUE TITLE
Want to set up lua-resty-waf on Kong API Gateway

### ISSUE THREAD

**Issue by gaganmurghai** [p0pr0ck5/FreeWAF#308]
Hello
I want to set up lua-resty-waf on Kong API Gateway, Kong is installed upon ubuntu 16.04. I tried to read the documentation listed in the README.md file but I was unable to get started. Can you please guide me how can go ahead and successfully install and set up lua-resty-waf on Kong API Gateway. Any help will be highly appreciated.
Thanks

**Comment by p0pr0ck5** [comment]
Hi,

Sorry for the delayed response. I actually have plans to write a Kong plugin to integrate this directly with Kong. Likely out towards the end of this year.

**Comment by shashankn91** [comment]
@p0pr0ck5 Is the plugin available now?

**Comment by p0pr0ck5** [comment]
@shashankn91 no, I haven't yet at time to create a plugin. Of course anyone is welcome to work on this if they like!

**Comment by zhenguang** [comment]
I've ported the plugin and added a waf block page 
https://github.com/zhenguang/kong-plugin-lua-resty-waf

I'm very new to lua and kong, so do pardon me if there are any mistake

Thanks
